src/__tests__/statistics.test.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { BrowserRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import axios from 'axios';
+import Statistics from '../pages/Statistics';
+import store from '../redux/store';
+import { getCovideStatistics } from '../redux/covid/covidSlice';
+
+it('Country  page renders correctly', () => {
+  const tree = renderer
+    .create(
+      <React.StrictMode>
+        <Provider store={store}>
+          <BrowserRouter>
+            <Statistics />
+          </BrowserRouter>
+        </Provider>
+      </React.StrictMode>,
+    )
+    .toJSON();
+  expect(tree).toMatchSnapshot();
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/227449654-8f6cf1e3-6528-4ac0-9886-ceb5edd5bf5a.png)

- [ ] I guess you have broken something so because of that all your tests are not passing 😨 Kindly make your tests pass because these tests tell if your application is working perfectly and ready for exceptional incidents 🤔 such as a break in the whole application or in a specific component 😊